### PR TITLE
fix: landing page layout issues on mobile

### PR DESCRIPTION
**Pull request by kranthicodes**
## Summary

This PR adds fix for layout zoom issue on mobile when user is on landing page
This also adds docs link to CTA on landing page

### CODE REVIEW

**Comment by pawanpaudel93** [comment]
1) Now it looks squeezed for non-mobile screens.

    image

2) Also, `Learn more` is missing the link to docs.

    image

File: src/components/Landing/Bounties.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default function Bounties() {
 
   function handleLearnMoreBtnClick() {
     trackGoogleAnalyticsEvent('Landing', 'Learn more button click', 'Learn more Button')
+    window.open('https://docs.protocol.land', '_blank')
```
Comment (7i7o) [inline src/components/Landing/Bounties.tsx:32]: Maybe we could send them directly into that section inside the docs?
[PL Docs - Create a new bounty](https://docs.protocol.land/working-with-issues/create-a-new-bounty)

**Review by 7i7o — CHANGES_REQUESTED** [review]
The middle card of the Bounties section says "Earn manage bounty submissions", was it supposed to be "Manage bounty submissions"?

File: src/components/Landing/Bounties.tsx (hunk 1 above)
Comment (pawanpaudel93) [reply src/components/Landing/Bounties.tsx:32]: Yes, i will update this link.

Comment (pawanpaudel93) [reply src/components/Landing/Bounties.tsx]: @7i7o I updated the link.

**Pull request merged**